tests/lifecycle/eventhub_cases_test.go

```diff
@@ -1,7 +1,13 @@
-// +build !unit
```

Comment:
The unit test failed for this. It should not be removed as the lifecycle test is not unit test.

Reply:
I returned the comment. Sorry, it was accidentally removed while cleaning other comments.

Reply:
no problem:)